x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -586,17 +663,30 @@ export class ManifestManager {
   private async listEndpointPolicies(page: number) {
     return this.packagePolicyService.list(this.savedObjectsClient, {
       page,
-      perPage: 20,
+      perPage: 100,
```

Comment:
can probably increase to at least 1000

Reply:
https://github.com/elastic/kibana/pull/160387/commits/07f122e199a58e825182d9fc2c26fe4c27f18809